templates/resources/config-network.properties

```diff
@@ -128,7 +125,7 @@ maxMessageSize = 1024
 [fork_heights]
 
 totalVotingBalanceCalculationFix = 528'000
-treasuryReissuance = 689761
+treasuryReissuance = 689'761
 
 [treasury_reissuance_transaction_signatures]
```

Comment:
also not needed, right?

Reply:
no, still needed to allow validation of nemesis transactions at that height

Reply:
discussed: I forgot NemesisSinkValidator is verifying them, so we need those for a bit longer